test/others/test_utils.py

```diff
@@ -931,3 +1061,125 @@ def test_upload_existing_eval_set(caplog):
+                        "name": "my-eval-run-1",
+                        "pipeline_name": "my-pipeline-1",
+                        "evaluation_set_name": "my-eval-set-1",
+                        "eval_mode": 0,
+                        "comment": "this is my first run",
+                        "debug": False,
+                        "tags": ["my-experiment-1"],
+                    }
+                )
+            ],
+        )
+
+        responses.add(
+            method=responses.GET,
+            url=f"{DC_API_ENDPOINT}/workspaces/default/eval_runs/my-eval-run-1",
+            json={
+                "created_at": "2022-05-24T12:13:16.445857+00:00",
+                "eval_mode": 0,
+                "eval_run_id": "17875c63-7c07-42d8-bb01-4fcd95ce113c",
+                "name": "my-eval-run-1",
+                "comment": "this is my first run",
+                "tags": ["my-experiment-1"],
+                "eval_run_labels": [],
+                "logs": {},
+                "eval_results": [
+                    {
+                        "node_name": "AnswerNode",
+                        "node_type": "answer_node",
+                        "isolated_exact_match": 1.0,
+                        "isolated_f1": 1.0,
+                        "integrated_exact_match": 0,
+                        "integrated_f1": 0,
+                    }
+                ],
+                "parameters": {
+                    "debug": False,
+                    "eval_mode": 0,
+                    "evaluation_set_name": "my-eval-set-1",
+                    "pipeline_name": "my-pipeline-1",
+                },
+                "status": 1,
+            },
+            status=200,
+        )
+
+        mock_prediction = {
+            "prediction_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+            "created_at": "2022-08-03T13:42:58.968Z",
+            "updated_at": "2022-08-03T13:42:58.968Z",
+            "eval_node_result_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+            "eval_mode": "Evaluation mode",
+            "query": "What?",
+            "context": "This",
+            "rank": 0,
+            "document_id": "0",
+            "filters": [{}],
+            "labels": [
+                {
+                    "label_id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+                    "query": "What?",
+                    "answer": "This",
+                    "answer_start": 0,
+                    "answer_end": 3,
+                    "meta": {},
+                    "context": "This",
+                    "external_file_name": "this_file.txt",
+                    "file_id": "3fa85f64-5717-4562-b3fc-2c963f66afa7",
+                    "state": "MATCHED",
+                    "candidates": None,
+                    "answer_exact_match": True,
+                    "f1": 1.0,
+                    "document_id_match": True,
+                    "answer_match": "Answer match",
+                    "context_similarity": "Context similarity",
```

Comment:
These should be numeric values.

Reply:
I will adjust these to the value type in DC.